ui/css/perf.css

```diff
@@ -664,3 +664,24 @@ li.pagination-active.active > button {
 .tests-table .rt-tr-group {
   flex: initial;
 }
+
+.alert-title-container {
+  width: 250px;
+}
+
+.alert-title-container .alert-title {
+  overflow-wrap: break-word !important;
```

Comment:
[question]hi, is important necessary here?

Reply:
Hey, whenever you are using things like react-strap, it is best to use !important to overwrite css rules, otherwise you might have surprises, so I prefer to use it here.